### PR TITLE
Redesign manager page

### PR DESCRIPTION
**Pull request by meriadec**
2018-04-13_1993x1262

### CODE REVIEW

File: src/components/ManagerPage/ManagerApp.js [hunk 1]
```diff
@@ -3,50 +3,65 @@
...
+  white-space: nowrap;
+`
+
 type Props = {
   name: string,
+  version: string,
   icon: string,
   onInstall: Function,
-  onUninstall: Function,
+  // onUninstall: Function,
```
Comment (loeck) [inline src/components/ManagerPage/ManagerApp.js]: ô ô

File: src/components/ManagerPage/ManagerApp.js [hunk 2]
```diff
@@ -3,50 +3,65 @@
...
-        <ActionBtn onClick={onInstall}>{'Install'}</ActionBtn>
-        <ActionBtn onClick={onUninstall}>{'Remove'}</ActionBtn>
+      <Box align="center">
+        <AppName>{name}</AppName>
+        <Text ff="Open Sans|Regular" fontSize={3} color="grey">
+          {version}
+        </Text>
       </Box>
+      <Button outline onClick={onInstall}>
+        {'Install'}
```
Comment (loeck) [inline src/components/ManagerPage/ManagerApp.js]: Translation ? :o

File: src/components/ManagerPage/AppsList.js [hunk 3]
```diff
@@ -146,6 +150,19 @@ class AppsList extends PureComponent<Props, State> {
       </List>
     )
   }
+
+  render() {
+    return (
+      <Box flow={6}>
+        <Box>
+          <Box mb={4} color="dark" ff="Museo Sans" fontSize={6}>
+            {'All apps'}
```
Comment (loeck) [inline src/components/ManagerPage/AppsList.js]: Translation

File: src/components/ManagerPage/ManagerApp.js (hunk 1 above)
Comment (meriadec) [reply src/components/ManagerPage/ManagerApp.js]: yeah.. we only have "Install" button now, but I'm pretty sure the uninstall will come in the future in the designs :) so I keep that here

**Pull request merged**